Stratis.Bitcoin.Features.Wallet/WalletManager.cs

```diff
@@ -591,9 +591,15 @@ public void RemoveBlocks(ChainedBlock fork)
                 var allAddresses = this.keysLookup.Values;
                 foreach (var address in allAddresses)
                 {
+					// remove all the UTXO that have been reorged. 
```

Comment:
Tabs, you are adding tabs to the code.
Also inline comment style pls.
Here and below as well (both tabs and comment style)